crowdsec-docs/unversioned/troubleshooting/issue_se_offline.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Security Engine Offline
-id: issue_security_engine_offline
+id: issue_se_offline
```

Comment:
- Enrollment revoked or pending: Engine enrollment was removed from the Console or is awaiting approval. -> not possible
- Enrollment revoked or pending -> kill
- Console connectivity issues -> add cscli capi status
- Re-enroll the engine in the Console -> kill
-